packages/cli/src/services/hooks.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Service } from 'typedi';
+import { UserService } from '@/services/user.service';
+import type { AssignableRole, User } from '@/databases/entities/User';
+
+/**
+ * Exposes functionality to be used by the cloud BE hooks.
+ * DO NOT DELETE any of the methods without making sure this is not used in cloud BE hooks.
+ */
+@Service()
+export class HooksService {
+	constructor(private userService: UserService) {}
+
+	/**
+	 * Invites users to the instance.
+	 * This method is used in the cloud BE hooks, to invite members during sign-up
+	 */
+	async inviteUsers(owner: User, attributes: Array<{ email: string; role: AssignableRole }>) {
+		return await this.userService.inviteUsers(owner, attributes);
```

Comment:
could you please add some tests?

Reply:
As I mention in the description, the downstream service (UserService.inviteUsers) has tests, and this is a wrapper on top. Should we clone that test?

Reply:
maybe a unit test here to make sure that invite user gets called correctly..

Reply:
What if instead we add instead a unit test checking that the method exists? `expect('inviteMembers' in userService).toBe(true)` Auto-renaming via IDE would not change a test like this right?

Reply:
Instead of what @mutdmour suggested or instead of introducing the hooks service?

Reply:
I think auto rename would update the names. Not a bad idea though 👍🏽 I think we have enough warnings and abstraction now though.